python/cuml/cuml_accel_tests/test_estimator_proxy.py

```diff
@@ -629,3 +630,56 @@ def test_get_feature_names_out():
 
     # No host transfer required
     assert not hasattr(model._cpu, "n_features_in_")
+
+
+@pytest.fixture
+def metadata_routing():
```

Comment:
```suggestion
def enable_metadata_routing():
```
or even more explicit: `with_metadata_routing_enabled`

Reply:
I have a slight preference for the current spelling (and a preference to avoid another CI cycle), merging as is for now.